manage_fastapi/templates/project/{{ cookiecutter.folder_name }}/docker-compose.yaml

```diff
@@ -1,8 +1,16 @@
 version: "3.8"
 
+volumes:
+{% if cookiecutter.database == "Postgres" %}
+  {{ cookiecutter.snake_name }}_local_postgres_data: {}
+{% elif cookiecutter.database == "MySQL" %}
+  {{ cookiecutter.snake_name }}_local_mysql_data: {}
+{% endif %}
+
 services:
   app:
     build: .
+    container_name: {{ cookiecutter.snake_name }}_local_fastapi
```

Comment:
Can't we just call it "app"? :thinking:

Reply:
Sure, that makes sense. I'll modify it now